<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workRecDialog</class>
 <widget class="QDialog" name="workRecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Работа по задаче</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="m_contentEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Описание выполненной работы</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Дата:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="m_dateEdit">
      <property name="toolTip">
       <string>Дата выполненной работы</string>
      </property>
      <property name="minimumDate">
       <date>
        <year>2001</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>220</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Время:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_timeEdit">
      <property name="toolTip">
       <string>Время, затраченное на выполнение работы</string>
      </property>
      <property name="inputMask">
       <string>000:00</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_fromTimerButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>48</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Установить время по таймеру</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="MainResource.qrc">
     <normaloff>:/images/timer48.png</normaloff>:/images/timer48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_addFromTimerButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>48</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Добавить к текущему времени из таймера</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="MainResource.qrc">
     <normaloff>:/images/addTimer48.png</normaloff>:/images/addTimer48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MainResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>workRecDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>workRecDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
